src/openfaas/4.4.0/values.yamle

```diff
@@ -0,0 +1,139 @@
+functionNamespace: openfaas-fn
```

Comment:
Is the file name supposed to be values.yamle?

Reply:
It's generated by `sed`. I don't see it as a blocker. Would like to try and get this in for additional e2e testing within DOKS.